css/main.css

```diff
@@ -247,44 +253,56 @@ body {
   background-color: #fff;
 }
 .head_section1 {
+  margin: 0;
   width: 445px;
   height: 87px;
   font-size: 36px;
   line-height: 43px;
 }
 .block_for_section_1 {
-  margin-top: 140px;
-  margin-bottom: 102px;
+  padding-top: 140px;
+  padding-bottom: 102px;
 }
 .text_section_1 {
+  margin: 0;
   width: 445px;
   height: 115px;
   font-size: 16px;
   line-height: 1.4;
+  padding-top: 56px;
 }
 .video {
-  width: 540px;
+  width: 100%;
   height: 321px;
   border-radius: 16px;
+  outline: none;
```

Comment:
Why do you need outline: none for video? As I know video has no outline by default.

Reply:
Yes we need.
Because if we added "controls" for video and click on play button, we will have black border around the video screen.
With "outline:none" we won't have this effect.
Check it out, add a comment to outline:none, controls for video and click on play the video.

Reply:
ok, but video with outline doesn't look like a bug to me.